src/mbgl/style/conversion/function.cpp

```diff
@@ -356,6 +356,17 @@ static optional<std::map<double, std::unique_ptr<Expression>>> convertStops(type
     return { std::move(stops) };
 }
 
+static void omitFirstStop(std::map<double, std::unique_ptr<Expression>>& stops) {
+    double min = std::numeric_limits<double>::infinity();
```

Comment:
Why `inf` is required? would `std::numeric_limits<double>::max()` suffice?

Reply:
The end result would be the same if `infinity` was changed to `max` even though the control flow would be slightly different in the case the first visited stop has a value of `max`. So `inf` is not required.

Do you have any specific concern with the use of infinity?

Reply:
IMO, using `infinity` implies need for the specific operations that it brings (i.e. inf + inf =inf, inf - inf = NaN), otherwise it looks to me a bit overkill :)

Reply:
Fair point. I'll change to `max`.